docs/content/introduction.rst

```diff
@@ -0,0 +1,17 @@
+Introduction
+============
```

Comment:
I would add a general introduction of what Cosmos-Academy is and paragraph on the general value proposition/mission of Cosmos/Tendermint